Ad performance by conversion type blog (#458)
readthedocs/ethicalads.io: content/posts/2024-best-performing-ads-by-conversion-type.md

Title: Ad Performance by Conversion Action: Do Developers Like Ebooks?
Date: October 29, 2024
description: As a marketer, should your ads link directly to a lead-gen form? Produce a gated resource like a webinar or ebook? Or go for the long sale and link to a blog with details about your product?
tags: advertisers, marketing, data-exploration
authors: David Fischer
image: /images/posts/2024-ad-conversion-type-billboard-image.jpg
image_credit: <span>Created with <a href="https://chatgpt.com/" title="advertising that knows what you think, billboards, future in an art sketch or watercolor style">ChatGPT</a></span>

> Digital marketing is all about generating conversions,
> getting website visitors to take a specific action on their site.
> Along with metrics like an ad's click-through rate (CTR),
> the conversion rate, the rate at which site visitors take that specific action,
> is one of the most important metrics in digital advertising.
> 
> The action that an advertiser wants somebody to take varies
> from signing up for a free trial, reading a blog,
> or filling out a lead generation form.
> The ad itself primes visitors on what to expect  on the landing page, with terms like "start free", "free ebook", "learn more", or "sign up".
> 
> Historically, we've dissuaded advertisers from linking to ebooks and webinars,
> especially gated ones, because they tend to have a lower CTR
> and advertisers have told us they don't convert as well.
> While CTOs and marketers may read ebooks at a higher rate, developers and individual contributors,
> who make up the bulk of our ad network don't seem to like them as much.
> In the rest of this post, we aim to back up our guidance with hard data from our own ad network.

Author: This section is a bit verbose. Here's a possibility to shorten it:

```suggestion
Historically, we've dissuaded advertisers from linking to ebooks and webinars,
especially gated ones, because they tend to have a lower click-through rate (CTR)
and advertisers have told us they don't convert as well.
While CTOs and marketers may read ebooks at a higher rate, developers and individual contributors,
who make up the bulk of our ad network don't seem to like them as much.

Ad landing pages vary by the type of conversion the marketer is looking for whether signing up for a free trial, putting in a credit card or
filling out a lead form for a resource like a webinar.
The ad itself primes visitors on what to expect on the landing page, with terms like "start free", "free ebook", "learn more", or "sign up".
In the rest of this post, we aim to back up our guidance with hard data from our own ad network on what types of ad conversions work best for developers.
```

Reviewer: Yea, looks a lot more direct :+1:

> Title: Ad Performance by Conversion Action: Do Developers Hate Ebooks?

Reviewer: ```suggestion
Title: Ad Performance by Conversion Action: Do Developers Like Ebooks?
```

Date: October 29, 2024
description: As a marketer, should your ads link directly to a lead-gen form? Produce a gated resource like a webinar or ebook? Or go for the long sale and link to a blog with details about your product?
tags: advertisers, marketing, data-exploration
authors: David Fischer
image: /images/posts/2024-cookieless-advertising.jpg
image_credit: <span>Created with <a href="https://openjourney.ai/" title="creepy ad targeting, invasive advertising, personalized advertising, ads that know what you think, ads that follow you around the internet, watercolor style">OpenJourney</a></span>

Digital marketing is all about generating conversions,

> getting website visitors and prospective customers to take a specific action on their site.

Reviewer: ```suggestion
getting website visitors to take a specific action on their site.
```

Along with metrics like an ad's click-through rate (CTR),
the conversion rate, the rate at which site visitors take that specific action,

> is one of the most important metrics in digital marketing and advertising.

Reviewer: ```suggestion
is one of the most important metrics in digital advertising.
```

> The action that an advertiser might want somebody to take on their site varies
> from signing up for a free trial of their product, reading a blog about the product,

Reviewer: ```suggestion
The action that an advertiser wants somebody to take varies
from signing up for a free trial, reading a blog,
```

or filling out a lead generation form.

> The ad itself frequently primes visitors on what to expect when they click an ad
> with terms like "start free", "free ebook", "learn more" or "sign up".

Reviewer: ```suggestion
The ad itself primes visitors on what to expect  on the landing page, with terms like "start free", "free ebook", "learn more", or "sign up".
```

We should probably have a style guide on the oxford comma. I always use them, and at least tech writers care a lot about them, probably not marketing as much.

Author: I'm pro-Oxford comma generally and I'm fine using it.

> Historically, we've dissuaded advertisers from linking to ebooks and webinars,
> especially gated ones, because we have seen lower CTR
> and advertisers have told us they haven't converted as well as alternatives
> (as a privacy focused ad network, we leave tracking conversions to advertisers).

Reviewer: ```suggestion
Historically, we've dissuaded advertisers from linking to ebooks and webinars,
especially gated ones, because they tend to have a lower CTR
and advertisers have told us they don't convert as well.
```

While CTOs and marketers may read ebooks at a higher rate, developers and individual contributors,
who make up the bulk of our ad network don't seem to like them as much.
In the rest of this post, we aim to back up our guidance with hard data from our own ad network.

## Types of ad conversions

A few months back, we created [guide to the kind of ad images that performed best]({filename}../pages/creatives-that-convert.md) across EthicalAds
and we saw that text-heavy ad images were among the lowest performers.
This time, we categorized all the ads shown on EthicalAds for the last month
by the desired marketing action and compared it with the click-through rate.

> * **Signup or lead form:** The ad linked to a page where a form was the primary above the fold action.

Reviewer: Are there lead forms without gated resources? I might do:

```suggestion
* **Signup form:** The ad linked to a page where a form was the primary above the fold action, generally with a free trial. 
```

* **Gated resource:** The ad linked to a lead-gated resource like an ebook or webinar but the visitor was required to submit their name and email to get access.
* **Blog:** The ad linked to a blog post explaining a product or a feature of the advertiser's product. Arguably, this is similar to public resource, but we broke it out despite some sample size issues.

Across our network, this was about 23 million paid ad impressions and about 32,000 paid clicks
and each conversion type except for Blogs had at least one million impressions in the month of data.

<div class="postimage text-center">
  <img class="w-100 shadow-lg" src="{static}../images/posts/2024-ad-ctr-conversion-type.png" alt="Ad CTR by Conversion Type (Sept-Oct 2024)">
  <p><small>Ad CTR by Conversion Type (Sept-Oct 2024)</small></p>
</div>

## Developers don't want to read your ebook

OK, the headline on this section is perhaps a little too strong.
However, I think it's fair to say after looking at the data
that developers on our network do not want to read ebooks especially lead-gated ones.

> Tailoring ad campaigns to the specific audience of the network is an important part
> of making sure ads perform well.

Reviewer: ```suggestion
Tailoring ad campaigns to developers is an important part of making sure ads perform well.
```

Everyone is busy and developers are no exception.
However, developers **are interested** in reading about products and services that solve their problems.
While some takeaways that the data points to may be specific to EthicalAds,
some of this is going to apply to anybody doing developer marketing:

* Developers are much more willing to give their email for a product trial or a free tier
  than they are for gated content like an ebook or webinar.
* Taking a developer directly to a product landing page or homepage when they click on an ad
  is fine especially if the page's primary target audience is already developers.
* For developer resources like webinars or ebooks,
  make sure this content is public and doesn't require a signup.
  A call to action of "signup" that isn't for a free trial is likely a red flag.

## We want your campaign to succeed

> One thing that separates a niche network like us from the huge ad players,
> is that advertisers are always a quick email away from reaching a real person
> who knows developer marketing on EthicalAds and can offer a little assistance
> in running a successful campaign.

Reviewer: ```suggestion
One thing that separates a niche network like us from the huge ad players,
is that advertisers are always a quick email away from reaching a real person
who knows developer marketing and can offer a little assistance
in running a successful campaign.
```
